### PR TITLE
Add tests for further test coverage

### PR DESCRIPTION
**Pull request by cyqjoseph**
Minor checkstyle improvements as well

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -62,6 +65,14 @@ public class CommandTestUtil {
     public static final EditCommand.EditPersonDescriptor DESC_AMY;
     public static final EditCommand.EditPersonDescriptor DESC_BOB;
 
+    public static final String VALID_BOOKING_DESCRIPTION = "Test Booking";
+    public static final String VALID_START_TIME = "2023-12-31 19:00";
+    public static final String VALID_END_TIME = "2023-12-31 23:00";
+
+    public static final String BOOKING_DESC_TEST = PREFIX_DESCRIPTION + VALID_BOOKING_DESCRIPTION;
+    public static final String START_TIME_DESC_TEST = " " + PREFIX_START_TIME + VALID_START_TIME;
+    public static final String END_TIME_DESC_TEST = " " + PREFIX_END_TIME + VALID_END_TIME;
```
Comment (gauravuj) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:74]: Good job on fixing these test cases!

File: src/test/java/seedu/address/logic/commands/ThemeCommandTest.java
Comment (gauravuj) [inline src/test/java/seedu/address/logic/commands/ThemeCommandTest.java:1]: Definetly needed!

File: src/test/java/seedu/address/logic/parser/ThemeCommandParserTest.java
Comment (gauravuj) [inline src/test/java/seedu/address/logic/parser/ThemeCommandParserTest.java:1]: Awesome!

**Review by gauravuj — APPROVED** [review]
Overall, great work! Thanks for adding these test cases

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (ChrysanthemumT) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:74]: nice!

File: src/test/java/seedu/address/logic/parser/BookCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        // Missing start time
+        String userInputMissingStartTime = BOOKING_DESC_TEST + END_TIME_DESC_TEST;
+        assertParseFailure(parser, userInputMissingStartTime,
+                           String.format(MESSAGE_INVALID_COMMAND_FORMAT, BookCommand.MESSAGE_USAGE));
+
+        // Missing end time
+        String userInputMissingEndTime = BOOKING_DESC_TEST + START_TIME_DESC_TEST;
+        assertParseFailure(parser, userInputMissingEndTime,
+                           String.format(MESSAGE_INVALID_COMMAND_FORMAT, BookCommand.MESSAGE_USAGE));
+    }
```
Comment (ChrysanthemumT) [inline src/test/java/seedu/address/logic/parser/BookCommandParserTest.java:33]: Thanks for adding this!

**Pull request merged**